server/server.lua

```diff
@@ -6,5 +6,21 @@ end)
 
 RegisterNetEvent("qbx-customs:server:buyCart", function(cart)
     local src = source
+    
+    local plate = cart.plate
+    local mods = cart.mods
 
-end)
\ No newline at end of file
+    if not IsVehicleOwned(plate) then return end
+    exports.oxmysql:execute('UPDATE player_vehicles SET mods = ? WHERE plate = ?', {json.encode(mods), plate})
+    
+end)
+
+
+
+local function IsVehicleOwned(plate)
```

Comment:
```suggestion
local function isVehicleOwned(plate)
```
camelCase local functions